.gitignore

```diff
@@ -1,14 +1,36 @@
+#built application files
+*.apk
```

Comment:
I guess Remove this thing for the apk to be included in the file

Reply:
No. APK should not be included in the repository. It'll available from `releases`.

Reply:
One more thing I forget to update the styles to NoActionBar please update that after merging

Reply:
Yeah okay!